hw3/03-react/game-of-thrones/src/views/Search.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import axios from "axios";
+import "../App.css";
+import TextField from "@mui/material/TextField";
+import Autocomplete from "@mui/material/Autocomplete";
+
+class Search extends Component {
+  state = {
+    characters: [],
+    filteredCharacter: {},
+  };
+  componentDidMount() {
```

Comment:
I am unfamiliar with this function, will look into it. Seems like it would be quite useful if it is just a quality check

Reply:
Thanks for your comment! `componentDidMount()` is a React lifecycle method that gets called after a component has been mounted in the DOM. It's commonly used to initialize component state or fetch data.